packages/keyring-controller/src/KeyringController.ts

```diff
@@ -187,28 +213,38 @@ export class KeyringController extends BaseController<
       setSelectedAddress: PreferencesController['setSelectedAddress'];
       setAccountLabel?: PreferencesController['setAccountLabel'];
     },
-    config?: Partial<KeyringConfig>,
-    state?: Partial<KeyringState>,
+    messenger: KeyringControllerMessenger,
+    config?: KeyringControllerConfig,
+    state?: Partial<KeyringControllerState>,
   ) {
-    super(config, state);
+    super({
+      name,
+      metadata: {
+        vault: { persist: false, anonymous: false },
+        isUnlocked: { persist: false, anonymous: false },
```

Comment:
We do include this flag as part of our Sentry state data, for debugging purposes. I don't think it can be considered as PII
```suggestion
        isUnlocked: { persist: false, anonymous: true },
```